lang/cs/Org.Apache.REEF.Utilities/Org.Apache.Reef.Utilities.DotNet.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AssemblyName>Org.Apache.Reef.Utilities</AssemblyName>
+    <OutputPath>..\bin\$(AssemblyName)</OutputPath>
```

Comment:
Yes, I added a build.DotNet.props file to mirror the previous file. Added additional properties to this file. I also added a solution file that we can build from. This too sits side-by-side the other project file.